tests/test_stac.py

```diff
@@ -0,0 +1,40 @@
+import os
+from tempfile import TemporaryDirectory
+import unittest
+
+import pystac
+from stactools.package import stac
+
+
+class StacTest(unittest.TestCase):
+    def test_create_collection(self):
+        # Write tests for each for the creation of a STAC Collection
+        with TemporaryDirectory() as tmp_dir:
+            # Create the STAC Collection...
+            stac.create_collection()
+
+            # Test that it has been created and validate
+            jsons = [p for p in os.listdir(tmp_dir) if p.endswith(".json")]
+            self.assertEqual(len(jsons), 1)
+
+            collection_path = os.path.join(tmp_dir, jsons[0])
+
+            collection = pystac.read_file(collection_path)
+
+            collection.validate()
+
+    def test_create_item(self):
+        # Write tests for each for the creation of STAC Items
+        with TemporaryDirectory() as tmp_dir:
+            # Create the STAC Item...
+            stac.create_item()
+
+            # Test that it has been created and validate
+            jsons = [p for p in os.listdir(tmp_dir) if p.endswith(".json")]
+            self.assertEqual(len(jsons), 1)
+
+            item_path = os.path.join(tmp_dir, jsons[0])
+
+            item = pystac.read_file(item_path)
+
+            item.validate()
```

Comment:
I'm personally in favor of asserting what the item should look like in this unit test, e.g. https://github.com/stactools-packages/threedep/blob/main/tests/test_stac.py#L13-L110. To me, it's a declarative way of saying "this is how I expect this known COG asset (or whatever) to be turned into a STAC item)". So maybe if we give a small skeleton of that (e.g. with an `self.assertEquals(item.id, "my-great-id")` and one or two more) we could encourage that.